src/Dev/Install/InstallRequirements.php

```diff
@@ -227,8 +227,9 @@ public function findWebserver()
     /**
      * Check everything except the database
      */
-    public function check()
+    public function check($originalIni)
```

Comment:
It's still not going to be strictly semver, but to make this non-breaking, can you assign this in a separate setter instead of adding a method argument? E.g. `setOriginalIni($ini)`.

Reply:
Or just give it a default value? `$originalIni = true`?

Reply:
derp, ignore me, it's not a bool